lib/ansible/modules/network/aci/aci_static_binding_to_epg.py

```diff
@@ -38,17 +38,18 @@
     description:
     - The name of the end point group.
     aliases: [ epg_name ]
-  encap:
+  encap_id:
     description:
-    - The VLAN encapsulation for the EPG.
-    - This acts as the secondary encap when using useg.
-    aliases: [ encapsulation ]
-    choices: [ range from 1 to 4096 ]
-  primary_encap:
+    - The encapsulation ID associating the C(epg) with the interface path.
+    - This acts as the secondary encap_Id when using micro-segmentation.
```

Comment:
Is this mixed-case on purpose or a typo ?
So should we also do `C(encap_id)` instead ?

Reply:
I will fix this after the #35806 is merged so I can rebase this and make updates in one swoop.